### PR TITLE
COM-3714 - changing the API call to ForStorepick as we need to list all the location that have StorePick option

### PR DESCRIPTION
**Pull request by kibo-DeepakTajane**
As quotes should not depend on the Inventory, All Store Pick location should be listed.
Changed the API to ForStorePick, which will list all Store Pick location that are not disabled.

### CODE REVIEW

File: scripts/modules/b2b-account/edit-quote.js [hunk 1]
```diff
@@ -1025,34 +1015,36 @@ define([
                             "address": fulfillmentInfo.fulfillmentContact.address
                         }
                     }));
+            }
```
Comment (sagarraut0007) [inline scripts/modules/b2b-account/edit-quote.js]: Can you remove all the unnecessary changes from your PR. looks like some changes happened due to formatting but remove those changes? Only keep the changes which you've made.

File: templates/modules/b2b-account/quotes/edit-quote/product-picker-table.hypr.live [hunk 2]
```diff
@@ -103,7 +103,10 @@
         {% if item.fulfillmentMethod != "Digital" %}
           <select class="mz-l-formfieldgroup-halfsize" data-mz-value="fulfillmentMethod" data-mz-quote-item="{{ item.id }}" >
             <option {% if item.fulfillmentMethod=="Ship" %} selected {% endif %} value="Ship">Direct Ship</option>
```
Comment (sagarraut0007) [inline templates/modules/b2b-account/quotes/edit-quote/product-picker-table.hypr.live]: Can you also take the labels "Direct Ship" and "Pickup" from localizations JSON files?

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

File: scripts/modules/b2b-account/edit-quote.js (hunk 1 above)
Comment (kibo-DeepakTajane) [reply scripts/modules/b2b-account/edit-quote.js]: remove the formatting changes

File: templates/modules/b2b-account/quotes/edit-quote/product-picker-table.hypr.live (hunk 2 above)
Comment (kibo-DeepakTajane) [reply templates/modules/b2b-account/quotes/edit-quote/product-picker-table.hypr.live]: moved them to label file

File: labels/en-US.json [hunk 3]
```diff
@@ -340,6 +340,8 @@
   "friday": "Friday",
   "from": "from",
   "fromDeliveryLocation": "from location",
+	"fulfillmentDirectShip": "Direct Ship",
```
Comment (sagarraut0007) [inline labels/en-US.json]: fix indentation over here. And you also have to add this to another localization file as well apart from english.

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

File: labels/en-US.json (hunk 3 above)
Comment (kibo-DeepakTajane) [reply labels/en-US.json]: done

**Review by kibo-raghu — APPROVED** [review]
Changes Approved

File: labels/en-US.json (hunk 3 above)
Comment (sagarraut0007) [reply labels/en-US.json]: @kibo-DeepakTajane , Based on previous comment, you've missed adding these new labels into "de-DE.json". see my original comment.

**Review by sagarraut0007 — APPROVED** [review]
